.github/workflows/ci.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.6"
+          # - "3.6"
```

Comment:
updated!

Reply:
@mambelli, after following your suggestion, I replaced the completely ignored unit tests for 3.6 with 3.7 and unit tests for python 3.7 reported the following:
```
Requirement already satisfied: htcondor>=9.0.0 in /opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages (from decisionengine-modules==0.1.dev958+gab47dfc) (10.3.0)
Requirement already satisfied: structlog>=21.1.0 in /home/runner/.local/lib/python3.7/site-packages (from decisionengine-modules==0.1.dev958+gab47dfc) (23.1.0)
Requirement already satisfied: requests>=2.14.2 in /opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages (from decisionengine-modules==0.1.dev958+gab47dfc) (2.31.0)
Requirement already satisfied: urllib3>=1.26.2 in /opt/hostedtoolcache/Python/3.7.16/x64/lib/python3.7/site-packages (from decisionengine-modules==0.1.dev958+gab47dfc) (2.0.2)
Collecting bill-calculator-hep>=0.1.4 (from decisionengine-modules==0.1.dev958+gab47dfc)
  Using cached bill_calculator_hep-0.1.4-py3-none-any.whl (24 kB)
Requirement already satisfied: numpy>=1.19.5 in /home/runner/.local/lib/python3.7/site-packages (from decisionengine-modules==0.1.dev958+gab47dfc) (1.21.6)
INFO: pip is looking at multiple versions of decisionengine-modules to determine which version is compatible with other requirements. This could take a while.
ERROR: Ignored the following versions that require a different python version: 1.4.0 Requires-Python >=3.8; 1.4.0rc0 Requires-Python >=3.8; 1.4.1 Requires-Python >=3.8; 1.4.2 Requires-Python >=3.8; 1.4.3 Requires-Python >=3.8; 1.4.4 Requires-Python >=3.8; 1.5.0 Requires-Python >=3.8; 1.5.0rc0 Requires-Python >=3.8; 1.5.1 Requires-Python >=3.8; 1.5.2 Requires-Python >=3.8; 1.5.3 Requires-Python >=3.8; 2.0.0 Requires-Python >=3.8; 2.0.0rc0 Requires-Python >=3.8; 2.0.0rc1 Requires-Python >=3.8; 2.0.1 Requires-Python >=3.8
ERROR: Could not find a version that satisfies the requirement pandas>=1.5.3; python_version >= "3.7" (from decisionengine-modules) (from versions: 0.1, 0.2, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.6.0, 0.6.1, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.8.0, 0.8.1, 0.9.0, 0.9.1, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.0, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.19.2, 0.20.0, 0.20.1, 0.20.2, 0.20.3, 0.21.0, 0.21.1, 0.22.0, 0.23.0, 0.23.1, 0.23.2, 0.23.3, 0.23.4, 0.24.0, 0.24.1, 0.24.2, 0.25.0, 0.25.1, 0.25.2, 0.25.3, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5)
ERROR: No matching distribution found for pandas>=1.5.3; python_version >= "3.7"
Error: Process completed with exit code 1.
```

It seems like anything above pandas 1.4.0 strictly requires python 3.8 or higher. Am I reading that right?
If yes, do we simply change your suggestion to switch to 3.8 instead of 3.7?

Reply:
Good idea. I checked and EOL of python 3.7 is at the end of June, let's go w/ 3.8

Reply:
updated to 3.8